checkstyle-suppressions.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suppressions PUBLIC
+  "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+  "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+
+<suppressions>
```

Comment:
checkstyle.xml and checkstyle-suppressions.xml are copied from azure-sdk-for-java?

Reply:
Yes, disabled a lot and relaxed 120 line max. Does not have time to fix all.